library/include/borealis/core/touch/scroll_gesture.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+    Copyright 2021 XITRIX
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#pragma once
+
+#include <borealis/core/touch/pan_gesture.hpp>
+
+namespace brls
+{
+
+/*
+ * Scroll recognizer
```

Comment:
I'm fine with having the scrolling code in the touch code as long as there is a TODO that says to move it somewhere else if mouse gets properly implemented. Can you add it?

When this PR gets merged I'll replace the TODO by a proper issue on GH.